checklist.md

```diff
@@ -3,7 +3,7 @@
 * Try to avoid code duplication. Especially, when you are working with ResultSet.
   Move retrieving data from ResultSet into Entity object to a separate private method.
 
-* Don't make `books.title` UNIQUE. This is not required in this task.
```

Comment:
revert file

Reply:
Please, revert changes in checklist file. If you have some questions please ask in chat